pages/common/feroxbuster.md

```diff
@@ -0,0 +1,20 @@
+# feroxbuster
+
+> Brute-forces hidden paths on web servers and more.
+> More information: <https://github.com/epi052/feroxbuster>.
+
+- Discover directories and files that match in the wordlist with e[x]tensions and 100 [t]hreads and a random user [A]gent:
+
+`feroxbuster -u {{https://example.com/}} -w {{path/to/file}} -t 100 -x php,txt -A`
```

Comment:
```suggestion
- Discover specific directories and files that match in the wordlist with extensions and 100 threads and a random user agent:

`feroxbuster --url "{{https://example.com/}}" --wordlist {{path/to/file}} --threads {{100}} --extensions "{{php,txt}}" --random-agent`
```

- Optional change: `specific` may sound better.
- Optional change: mnemonics are redundant for long options.
- Optional change: long options available and can be used as they are easily memorable.
- Optional change: url and extensions can be quoted to protect strings from unwanted shell expansions.
- Optional change: placeholders for some options can be used.